src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Bau/BauCacheController.cs

```diff
@@ -61,9 +64,24 @@ public async Task<ActionResult> ClearPublicCacheTrees(ClearPublicCacheTreePathsV
                 await request.Paths
                     .ToAsyncEnumerable()
                     .ForEachAwaitAsync(
-                        path =>
-                            _publicBlobStorageService.DeleteBlob(BlobContainers.PublicContent, path)
-                    );
+                        
+                        async path =>
+                        {
+                            var allowedPath =
```

Comment:
I'll need to update some BAU doco somewhere to reference these new allowed values.  This was done as a way to break the dependency between AllMethodologiesCacheKey and PublicationTreeCacheKey with the Admin project, as a precursor to moving all public-facing caching code into the Content.Api and Data.Api projects themselves.

Reply:
Updated on the testing ticket